examples/react-native/types/env.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module '@env' {
+  export const Greeting: string;
```

Comment:
Can we use SCREAMING_SNAKE_CASE here as these will be constants?

Reply:
Yes!